images/toolbox/Dockerfile

```
@@ -1,3 +1,22 @@
+ARG wget2_tarball="wget2-2.1.0"
+
+FROM public.ecr.aws/lts/ubuntu:22.04_stable AS builder
+
```

If you set `-e` and whatnot in SHELL like you're already doing for the other stage below, you'll get more-helpful error messages if e.g. `apt-get-update` fails.

```suggestion
SHELL ["/bin/bash", "-euo", "pipefail", "-c"]

```